### PR TITLE
Add support for multiple workers to the signal collection

### PR DESCRIPTION
**Pull request by calebbrown**

### CODE REVIEW

File: cmd/collect_signals/main.go [hunk 1]
```diff
@@ -44,6 +50,71 @@ func init() {
...
+			logger.WithFields(log.Fields{
+				"status":      arle.Response.Status,
+				"headers":     arle.Response.Header,
+				"message":     arle.Message,
+				"retry_after": arle.RetryAfter,
+			}).Error("Abuse Error error...")
+		}
+		var er *gh.ErrorResponse
+		if errors.As(err, &er) {
+			f, err2 := os.Create("fail.html")
```
Comment (oliverchang) [inline cmd/collect_signals/main.go:82]: nit: do we actually need `err2`? We don't need the original `err` in this scope right?

File: cmd/collect_signals/main.go [hunk 2]
```diff
@@ -134,58 +218,20 @@ func main() {
...
+		repos <- u
 	}
 	if err := scanner.Err(); err != nil {
 		logger.WithFields(log.Fields{
 			"error": err,
 		}).Error("Failed while reading input")
 		os.Exit(2)
 	}
+	// Close the repos channel to indicate that there is no more input.
+	close(repos)
```
Comment (oliverchang) [inline cmd/collect_signals/main.go:231]: Should this be in a defer instead?

**Review by oliverchang — APPROVED** [review]
nice!

File: cmd/collect_signals/main.go (hunk 2 above)
Comment (calebbrown) [reply cmd/collect_signals/main.go:205]: No. It needs to happen before `wait()` defer would force it to happen afterwards.

At some point I will need to gracefully terminate other workers on a SIGTERM or a dead worker (unrecoverable error), but that will need more improvements.

File: cmd/collect_signals/main.go (hunk 1 above)
Comment (calebbrown) [reply cmd/collect_signals/main.go]: This was debugging code. I've removed it.

**Pull request merged**